hubclient/rapid-scans-client.go

```diff
@@ -64,3 +72,40 @@ func (c *Client) UploadBdioFiles(bdioUploadEndpoint string, bdioContents []strin
 
 	return nil
 }
+
+func (c *Client) PollRapidScanResults(rapidScanEndpoint string, intervalInSeconds int, timeoutInSeconds int) (error, *hubapi.RapidScanResult) {
```

Comment:
Just pass the `time.Duration` - it is less confusing, no need for these name games with xxxInSeconds etc.